docs/source/upgrade_to_newest_version.md

```diff
@@ -0,0 +1,186 @@
+# Considerations for getting to v2.0
+
+## Chaincode lifecycle
+
+For more information about the new chaincode lifecycle, check out [Chaincode for operators](./chaincode4noah.html).
+
```

Comment:
Could we add a little more text here around what is the new cc lifecycle/what changed? "Fabric 2.0 introduces a new chaincode lifecycle process that ...."

Reply:
Added an additional sentence. In general I would rather avoid too much summarizing when someone should really go read the doc I'm linking to.